doc/examples/debug/lib/main.dart

```diff
@@ -1,13 +1,12 @@
 import 'package:flame/components.dart';
 import 'package:flame/game.dart';
-import 'package:flame/flame.dart';
 
 import 'package:flutter/material.dart' hide Image;
 
 import 'dart:ui';
 
 void main() async {
-  Flame.initializeWidget();
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
I thought we didn't need this anymore

Reply:
We don't need initialDimensions, this is still needed when we load assets before the runApp